dali/operators/reader/loader/file_label_loader.cc

```diff
@@ -127,9 +127,11 @@ void FileLabelLoader::ReadSample(ImageLabelWrapper &image_label) {
     }
     image_label.image.Resize({image_size});
     // copy the image
-    current_image->Read(image_label.image.mutable_data<uint8_t>(), image_size);
+    Index ret = current_image->Read(image_label.image.mutable_data<uint8_t>(), image_size);
+    DALI_ENFORCE(ret == image_size, make_string("Failed to read ", image_pair.first, " file."));
```

Comment:
```suggestion
    DALI_ENFORCE(ret == image_size, make_string("Failed to read ", image_pair.first));
```

Reply:
```suggestion
    DALI_ENFORCE(ret == image_size, make_string("Failed to read file: ", image_pair.first);
```
I'd keep the word "file" somewhere and add quotes and/or colon. What you suggest may lead to very cryptic messages, depending on the file name - here are some examples:
"Failed to read registry"
"Failed to read memory"

Reply:
Done